src/Symfony/Bundle/WebProfilerBundle/Profiler/TemplateManager.php

```diff
@@ -67,13 +67,15 @@ public function getName(Profile $profile, $panel)
      *
      * @param Profile $profile
      *
-     * @return array
+     * @return Twig_Template[]
+     *
+     * @deprecated not used anymore internally
```

Comment:
Agreed, but not in Symfony 2.7. This must be added after merging this to `master`.